transactions/reset-epoch/2023/feb-23/arguments.json

```
@@ -0,0 +1,26 @@
+[
+	{
+		"type":"UInt64",
+		"value":"59"
+	},
+	{
+		"type":"String",
+		"value":"6bbf87f18f42db1de550f29e915e5986"
+	},
+	{
+		"type":"Optional",
+		"value":null
+	},
+	{
+		"type":"UInt64",
+		"value":"0"
+	},
+	{
+		"type":"UInt64",
+		"value":"426399"
+	},
+	{
+		"type":"UInt64",
+		"value":"447719"
+	}
+]
```

These are placeholders only - the real values need to be filled in after the spork. 
```suggestion
[
	{
		"type":"String",
		"value":"invalid argument which will cause the transaction creation to fail - please replace me after the spork"
	}
]
```

---

Thanks, I removed those so it is clear we need to fill them in